modules/4337/contracts/Safe4337Module.sol

```diff
@@ -211,6 +211,43 @@ contract Safe4337Module is IAccount, HandlerContext, CompatibilityFallbackHandle
         operationHash = keccak256(operationData);
     }
 
+    /**
+     * @dev Checks if the signatures length is correct and does not contain additional bytes. The function does not check the integrity of the signature encoding.
```

Comment:
nit: Can you keep the line length down? I can't remember if we use 100 or 120 characters (take a look at other comments).

Reply:
```suggestion
     * @dev Checks if the signatures length is correct and does not contain additional bytes. The function does not check the integrity of the signature encoding, as this is expected to be checked by the {Safe} implementation of {checkSignatures}.
```